### ISSUE TITLE
[FEATURE] Loading/Saving

### ISSUE THREAD

**Issue by nevercast** [JimAllanson/lifejs#2]
It would be fantastic if I could pause the simulation and export the current state of the simulation to some serialized string that I can re-import at a later time.

**Comment by rwill128** [comment]
Agreed. I just discovered this project but I will take a look at it this week and see what might be involved in doing that.
